FinHubSA/information-access
# feat: 141 Mobile compatible status bar

# Pull Request

## Scope
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_workitems/edit/141

## Work Done

Made status bar that shows number of search results returned. Status bar optimised for mobile to contain search filters if screen is less than 900 px. Components search filter separated from search bar for alignment in formatting. Navbar reorganised on introduction of status bar fucntionality. Drawer readdition of home page with removal of logo with home routing.

## Steps to Test
```npm run start```
Check for sizes less than 900px using developer tools. Note: most common minimum mobile size is 375 width.

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/136695212-e0b0f959-c636-4715-9513-74dbe516dc35.png)
![image](https://user-images.githubusercontent.com/80747408/136695328-eb4bdabe-323f-4c64-b8aa-7f7dad4fda10.png)
